packages/@lightningjs/ui-components/src/components/TextBox/TextBox.js

```diff
@@ -233,6 +234,7 @@ export default class TextBox extends Base {
         h: this.h,
         y: this.style.offsetY,
         x: this.style.offsetX,
+        ...this.marqueeProps,
```

Comment:
lets move this destructure to the top of the object so we dont accidentally override any declared properties